### DISCUSSION TITLE
Event that rerender the component without browser refresh

### DISCUSSION THREAD

**Discussion by roeiboribor** [Power-Components/livewire-powergrid#853] [Q&A]
Hi good day, I would like to ask some questions regarding the passed variables from the parent to child (powergrid). I have an external date filters. Because I wanted to have a default date filters so that it won't load all the 100K+ records upon initial use. I am using the default $refresh event but the datatable is not fetching the updated query from datasource(). Is there other way to call that function again like "rerender" the component again? Thank you so much for this repo very helpful it saves lots of time developing sort, filter & search function.

**Comment by BenOussama180** [comment]
`        $this->emit('pg:eventRefresh-default');
`
